### PR TITLE
Bundle services with the cluster

### PR DESCRIPTION
**Pull request by gabriel-v**

### CODE REVIEW

File: templates/dnsmasq.nomad [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        args = ["--log-facility=-", "--conf-file=/etc/dnsmasq.conf"]
+        cap_add = ["NET_ADMIN"]
+        volumes = [
+          "local/dnsmasq.conf:/etc/dnsmasq.conf"
+        ]
+      }
+
+      template {
+        data = <<-EOF
+        bind-interfaces
```
Comment (mgax) [inline templates/dnsmasq.nomad]: Please add one level of indentation

File: templates/nomad.hcl [hunk 2]
```diff
@@ -31,6 +31,7 @@ client {
   }
   options {
     "fingerprint.blacklist" = "env_aws"
+    "docker.caps.whitelist" = "NET_ADMIN,CHOWN,DAC_OVERRIDE,FSETID,FOWNER,MKNOD,NET_RAW,SETGID,SETUID,SETFCAP, SETPCAP,NET_BIND_SERVICE,SYS_CHROOT,KILL,AUDIT_WRITE"
```
Comment (mgax) [inline templates/nomad.hcl]: That looks like a lot of capabilities, do we need all of them now?

File: templates/dnsmasq.nomad (hunk 1 above)
Comment (gabriel-v) [reply templates/dnsmasq.nomad]: The `-EOF` only removes whitespace up to that point.

File: templates/nomad.hcl (hunk 2 above)
Comment (gabriel-v) [reply templates/nomad.hcl]: We only need NET_ADMIN for dnsmasq, but the other ones are defaults listed in the nomad docs. If you disable any of the default ones, all your jobs will fail to run.

**Pull request merged**